package.json

```diff
@@ -19,6 +19,7 @@
     "build:interfaces": "node output/tools/generateContractInterfaces.js",
     "build": "npm run build:source && npm run lint && npm run build:contracts && npm run build:interfaces",
     "test:unit": "pytest tests -vv",
+    "test:unit:single": "pytest -vv",
```

Comment:
How is this different from `test:unit`?

Reply:
`test:unit` defaults to the "tests" directory, this allows us to pass in some tests. It's just there so you can iterate on a single unit test.